tests/feinterface/stokes.cc

```diff
@@ -0,0 +1,2542 @@
+/* ---------------------------------------------------------------------
```

Comment:
There is a lot of commented code in this test. What do we actually test here?

Reply:
Agreed. That is my fault for handing him a large and incomplete test code. Jiaqi, please remove from thi PR. We will submit additional tests later.